src/js/components/Profile.jsx

```diff
@@ -71,12 +82,20 @@ const Profile = (props: {
             handleHeadlineSubmit={handleHeadlineSubmit}
             headline={headline}
             handleImageSubmit={handleImageSubmit}
+            socialUrls={socialUrls}
+            handleSocialMediaUrlSubmit={handleSocialMediaUrlSubmit}
+            socialInputs={socialInputs}
+            handleSocialUrlChange={handleSocialUrlChange}
           />
         )}
       </div>
     );
   }
-  // otherwise just view a profile
+
+  // Otherwise just view a profile
+  const selectedProfile = filter(appProfiles, profile => profile.uid === uid);
```

Comment:
Nice touch!

Reply:
Thanks. Hope this doesn't introduce add complexity. Let me know how readable this is for you.